src/menu.cc

```diff
@@ -112,6 +112,16 @@ Menu::Menu() {
         </item>
       </section>
       <section>
+      <item>
+        <attribute name='label' translatable='yes'>_Toggle _Line _Comment</attribute>
+        <attribute name='action'>app.source_comments_toggle</attribute>
+      </item>
+      <item>
+        <attribute name='label' translatable='yes'>_Toggle _Block _Comment</attribute>
```

Comment:
In my opinion we should keep it simple. There is added complexity in adding block comments, not only do we have to add more language exceptions, but we have cases where this will result in an error:
```c++
  /*
  /*
  */
  */
```

While line comments always works:
```c++
  /*
  //
  //
  // */
```

Emacs also always add line comments only as far as I know. Finally, you would have one function for adding comments, instead of two.